src/main/scala-sbt-0.13/org/allenai/plugins/Compat.scala

```diff
@@ -1,22 +0,0 @@
-package org.allenai.plugins
```

Comment:
These two Compat files won't be necessary anymore because we will no longer be publishing across sbt versions?

Reply:
Correct